<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>client</class>
    <widget class="QWidget" name="client">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>448</width>
                <height>556</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <spacer name="horizontalSpacer">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Maximum</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="0" column="1">
                <widget class="QSplitter" name="splitter">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <widget class="QWidget" name="layoutWidget">
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <widget class="QLabel" name="gameLabel">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="pixmap">
                                        <pixmap>snakes.gif</pixmap>
                                    </property>
                                    <property name="scaledContents">
                                        <bool>false</bool>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QVBoxLayout" name="verticalLayout">
                                    <item>
                                        <spacer name="verticalSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Vertical</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>20</width>
                                                    <height>40</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="playerNameLabel">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string>Player Name:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="playerNameLine"/>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QPushButton" name="hostButton">
                        <property name="text">
                            <string>HOST YOUR GAME</string>
                        </property>
                    </widget>
                    <widget class="QTableWidget" name="avaliableGamesTable">
                        <property name="maximumSize">
                            <size>
                                <width>371</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="sizeAdjustPolicy">
                            <enum>QAbstractScrollArea::AdjustIgnored</enum>
                        </property>
                        <property name="editTriggers">
                            <set>QAbstractItemView::NoEditTriggers</set>
                        </property>
                        <property name="columnCount">
                            <number>4</number>
                        </property>
                        <attribute name="horizontalHeaderMinimumSectionSize">
                            <number>4</number>
                        </attribute>
                        <attribute name="verticalHeaderVisible">
                            <bool>false</bool>
                        </attribute>
                        <column>
                            <property name="text">
                                <string>Master</string>
                            </property>
                            <property name="font">
                                <font>
                                    <bold>true</bold>
                                </font>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Players</string>
                            </property>
                            <property name="font">
                                <font>
                                    <bold>true</bold>
                                </font>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Size</string>
                            </property>
                            <property name="font">
                                <font>
                                    <bold>true</bold>
                                </font>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Food</string>
                            </property>
                            <property name="font">
                                <font>
                                    <bold>true</bold>
                                </font>
                            </property>
                        </column>
                    </widget>
                </widget>
            </item>
            <item row="0" column="2">
                <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Maximum</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="1">
                <spacer name="verticalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Maximum</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="2" column="1">
                <widget class="QPushButton" name="modeButton">
                    <property name="text">
                        <string>MODE: NORMAL</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
